pkg/monitor/worker.go

```diff
@@ -127,10 +134,32 @@ func (mon *monitor) changefeed(ctx context.Context, baseLog *logrus.Entry, stop
 				break
 			}
 
+			baseLog.Debugf("subscriptions changefeed page was %d docs", subs.Count)
+
 			mon.mu.Lock()
 
 			for _, sub := range subs.SubscriptionDocuments {
-				mon.subs[sub.ID] = sub
+				id := strings.ToLower(sub.ID)
```

Comment:
Could you add a comment why we need lower case the subscription ID?

Reply:
It's normalization for database